site/source/tutorials/vdom_abstractions.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: false
+title: vDOM Abstractions - w and v in the d module
+overview: Abstracting the DOM allows us to create presentation components programatically. Dojo 2 provides a layer of abstraction between the DOM and its virtual counterpart as it exists in JavaScript and a layer of abstraction between this virtual DOM and Dojo 2 components - using a common syntax in a common module.
```

Comment:
as it exists in JavaScript and a layer of abstraction -> as it exists in JavaScript as well as a layer of abstraction